src/extension/utils.ts

```diff
@@ -5,21 +5,28 @@ import cp from 'node:child_process'
 import vscode, { FileType } from 'vscode'
 
 import { CONFIGURATION_SHELL_DEFAULTS } from '../constants'
+import { WasmLib } from '../types'
 
 import executor from './executors'
 import { ENV_STORE, DEFAULT_ENV } from './constants'
 
 const HASH_PREFIX_REGEXP = /^\s*\#\s*/g
 
+export function getMetadata(cell: vscode.NotebookCell) {
+  const metadata: WasmLib.Metadata = cell.metadata
+  return metadata
+}
```

Comment:
Is there a need for this utility function? Is this for typing purposes?

Reply:
Mainly yes, it also came handy for testing. The type-casting in place seemed unnecessary verbose. I'm open to suggestions if you want to push a "refactor" on top of my branch.

Reply:
Pushed https://github.com/stateful/vscode-runme/pull/95 without running the tests, I am in some meetings and will clean up after

Reply:
👍 sounds good